src/wp-includes/pluggable.php

```diff
@@ -1068,10 +1068,12 @@ function wp_set_auth_cookie( $user_id, $remember = false, $secure = '', $token =
 		 * Allows preventing auth cookies from actually being sent to the client.
 		 *
 		 * @since 4.7.4
+		 * @since 6.2.0 The `$user_id` parameter was added.
 		 *
-		 * @param bool $send Whether to send auth cookies to the client.
+		 * @param bool $send    Whether to send auth cookies to the client.
+		 * @param int  $user_id User ID. Zero when clearing cookies.
 		 */
-		if ( ! apply_filters( 'send_auth_cookies', true ) ) {
+		if ( ! apply_filters( 'send_auth_cookies', true, $user_id ) ) {
```

Comment:
Something else that occurred to me, is that it could be useful to know _what_ cookie is being set.

For example, one might want to know about the `$token` for the Session ID. Which made me question if matching the other filters in the function is worthwhile. 

```suggestion
		if ( ! apply_filters( 'send_auth_cookies', true, $user_id, $expire, $expiration, $token ) ) {
```

Reply:
See https://meta.trac.wordpress.org/browser/sites/trunk/common/includes/wporg-sso/wp-plugin.php?rev=12232&marks=631-654#L630 for where I've previously worked around the lack of context on `send_auth_cookies` through the usage of the `set_auth_cookie` filter and statics.

Reply:
Sounds good to me @dd32!